mesop/web/src/shell/shell.ts

```diff
@@ -257,6 +241,34 @@ export class Shell {
     userEvent.setResize(new ResizeEvent());
     this.channel.dispatch(userEvent);
   }
+
+  // Executes the scroll command if a key has been specified.
+  maybeExecuteScrollCommand() {
+    if (this.commandScrollKey) {
+      const targetElements = document.querySelectorAll(
+        `[data-key="${this.commandScrollKey}"]`,
+      );
+      if (!targetElements.length) {
+        console.error(
+          `Could not scroll to component with key ${this.commandScrollKey} because no component found`,
+        );
+        return;
```

Comment:
probably would be good to do this before the return:
`this.commandScrollKey = '';` 

this branch is already an edge case, but having the command scroll key somehow run in a future render loop when the the target element exists, seems like an even more bizzare edge case.

Reply:
Ah yeah good call.